src/components/BigNumber/bigNumberFormatUtils.ts

```diff
@@ -0,0 +1,25 @@
+import { BigNumberProps } from 'types';
```

Comment:
This needs the copyright

run `yarn copyrights` from the root folder and the script will add it to all files that need it.